README.md

```diff
@@ -122,6 +122,25 @@ hive.allow-drop-table=true
 hive.allow-rename-table=true
 ```
 
+#### Session properties per model
```

Comment:
What will happen if we setup session proerties in dbt profile and in dbt hooks, which would take precedence?

Reply:
`SET session = value;` should override the set value. This is working as expected, tested with `query_max_run_time`